upstairs/src/lib.rs

```diff
@@ -2264,6 +2254,10 @@ async fn looper(
         } else {
             tokio::time::sleep(Duration::from_secs(1)).await;
         }
+        // Get the specific information for the downstairs we will operate on.
```

Comment:
After every disconnect from a downstairs, the looper loop will reacquire the IP:Port for the downstairs and try to connect again.   We will probably put this somewhere else when
support for changing a downstairs comes in, but having it where it is now is a step in the
right direction.